admin/tests/integration/components/users/user-detail-personal-information/authentication-method_test.js

```diff
@@ -42,7 +52,13 @@ module('Integration | Component | users | user-detail-personal-information/authe
           );
 
           // then
-          assert.dom(screen.getByText('Adresse e-mail non confirmée')).exists();
+          assert
+            .dom(
+              screen
+                .getAllByRole('listitem')
+                .find((listItem) => listItem.textContent?.trim() === 'Adresse e-mail non confirmée')
+            )
+            .exists();
```

Comment:
ça passerait pas ça ? 
```suggestion
          assert
            .dom(
              screen
                .getByRole('listitem', name: { 'Adresse e-mail non confirmée'})
            )
            .exists();
```

Reply:
Malheureusement cela ne fonctionne pas. J'avais tenté le coup et j'ai du faire une recherche pour trouver une solution qui est celle-ci : https://stackoverflow.com/questions/63033144/unable-to-use-getbyrole-on-listitem-with-a-specific-name-rtl